### ISSUE TITLE
add ability assessment feature

### ISSUE THREAD

**Issue by jdigiac2** [ASU-CodeDevils/flameboi-slack-api#13]
Have flameboi assess new user's abilities and recommend resources based on user abilities. Also will help project managers to assess team-member skill levels which could allow them to give people tasks/guidance based on their level of expertise

Copied Text From Duplicate Card in Feature Integration Project 
Have flameboi assess new users' abilities as they are added with a short survey. The information gained can be used by project managers and other seasoned coders to help guide less experienced members through projects.  If the new member happens to be a beginner, flameboi could maybe make certain recommendations such as: websites to join (github or hackerrank), software to download, pertinent online learning resources. The same goes with higher levels of competency, flameboi could make recommendations like this based on the users skill level.

**Comment by bananabrann** [comment]
What's "assess new user's abilities?" Like, read their code and assign a level of competency, or user input? 
@jdigiac2

**Comment by jdigiac2** [comment]
Im thinking give them a short survey to figure out where they stand skills-wise, strengths and what their goals are. Flameboi could assign a beginner, intermediate or advanced (as an example, we could use different levels) string to a dictionary with their name name, or it could ask about the new user to self-asses proficiency with some commonly used languages and maybe send them some resources based on the results or store a report with strengths and weaknesses that project managers could use to make more informed decisions when assigning tasks.

**Comment by stucamp** [comment]
added this issue to the Feature Integration Project in lieu of the existing card.  Think it'll be easier for everyone to keep track of these if we're adding them as an Issue and attaching the issue to the "Feature Integration" project.  (Not directed at you Jesse! You did perfect! Just kinda thinking best MO to keep organized on my part!)

**Comment by bananabrann** [comment]
Threw this to officers chat, so far the consensus is that if this were to be implemented, it would need to be private... We want to avoid any sort of classes, tiers, languages, or any other sorting as to not exclude folks at any time. Projects and other recommendations can be posted publicly, but the level or score would need to not be, including to yourself.

Ex: I can do `/retake` or `/recommend` but cannot do something like `/score`

If that changes with more officers input, I'll update prompto!

**Comment by jdigiac2** [comment]
I agree 100% i definitely dont want to turn it into a ranking system, i just want to foster an environment where it is easier to utilize the experience and knowledge of our more advanced coders to mentor and help the members those of us who dont have as much experience. And, speaking for myself, i want to help on as many projects and get as much experience as I can, and I read through the files but i have no idea where to even begin to contribute. So i think something like that could help our organization to know where to begin when it comes to growing our members through projects and mentoring.

**Comment by jdigiac2** [comment]
If it seems like it will become an issue then we dont have to even save the info, just use the survey to make  reccomendations

**Comment by bananabrann** [comment]
No worries man --feel free to checkout a branch and work on it if you wanted, just wanted to bring it up because we've had a ranking system pop up in discussion before. Want to get the specs right before you work hard on something to implement
